src/transformState.js

```diff
@@ -47,6 +47,46 @@
  */
 function transformState(state, transforms) {
   // write code here
+  let objectTransforms = {};
+  // let objectProperties = {};
+  const fixedState = state;
+
+  for (const detail of transforms) {
+    objectTransforms = {
+      objectTransforms,
+      ...detail,
+    };
+
+    switch (objectTransforms.operation) {
+      case 'addProperties':
+        const addProperties = objectTransforms.properties;
+
+        for (const key in addProperties) {
+          fixedState[key] = addProperties[key];
+        }
+
+        break;
+      case 'removeProperties':
+        const keyOfDelete = Object.values(objectTransforms.properties);
+
+        for (const key of keyOfDelete) {
```

Comment:
```suggestion
        for (const property of objectTransforms.properties) {
```
це ж масив там зберігається

Reply:
ясно, зараз виправлю